.github/workflows/rust.yml

```diff
@@ -20,3 +20,5 @@ jobs:
       run: cargo build --verbose
     - name: Run tests
       run: cargo test --verbose
+    - name: Run Bench
+      run: cargo bench --verbose
```

Comment:
Not sure that we need to always run the benchmarks in CI, so adding `--no-run` here I think works to at least make sure they compile.

Reply:
I wanted to include the bench that way we can easily track performance regressions in PR without having to checkout the PR and running it locally.